### ISSUE TITLE
about the RSSN

### ISSUE THREAD

**Issue by ljc19940403** [JizhiziLi/GFM#17]
hello, thanks for your excellent work.I'm interested in the performance of the composite image with RSSN.Do you have the code about the composite image?I want to test the effect.（sorry，I didn't find it in the project）
Looking forward to your reply

**Comment by JizhiziLi** [comment]
Hi there, sorry we cannot release the code until the paper review is finished. But RSSN is described clearly in our paper and easy to implement:
1) Adopt BG-20k to serve as background images;
2) Add blur to the background randomly to simulate shallow depth-of-field;
3) Adopt denoise foreground, background, add Gaussian noise to the composite images.

All the parameters and procedure are described clearly in the paper, feel free to contact me if you have any further questions. :)

**Comment by ljc19940403** [comment]
> Hi there, sorry we cannot release the code until the paper review is finished. But RSSN is described clearly in our paper and easy to implement:
> 
> 1. Adopt BG-20k to serve as background images;
> 2. Add blur to the background randomly to simulate shallow depth-of-field;
> 3. Adopt denoise foreground, background, add Gaussian noise to the composite images.
> 
> All the parameters and procedure are described clearly in the paper, feel free to contact me if you have any further questions. :)

Thank you for your reply.I've finished the code.
Recently when I was learning about Salient Object Detection，I used composite images to augment the training set,
C= F × α + B × (1 − α).Some exceptions were found in the loss and test results,the final effect does not reach the expectation, and some results often appear low gray value of the shadow.Therefore, I would like to try using your scheme.Did you encounter the same scenario in your initial tests when you only use C= F × α + B × (1 − α)?
